packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx

```diff
@@ -76,6 +77,12 @@ export class SankeyChartBase extends React.Component<ISankeyChartProps, ISankeyC
       selectedNodes: new Set<number>(),
       shouldOverflow: false,
       isCalloutVisible: false,
+      emptyChart: !(
+        this.props.data &&
+        this.props.data.SankeyChartData &&
+        this.props.data.SankeyChartData.nodes.length > 0 &&
```

Comment:
Check with Ankit if a sankey chart with no links is a valid Sankey chart

Reply:
checked with Ankit, a Sankey chart with no links is not a valid Sankey Chart.